src/components/index/Promo.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import styled from '@emotion/styled'
+import { breakpoints } from '../../utils/variables'
+
+const StyledPromo = styled.div`
+    display: flex;
+    border: 2px solid #ebebeb;
+
+    &:nth-child(2n) {
+        flex-direction: row-reverse;
+
+        @media(max-width: ${breakpoints.md}) {
+            flex-direction: column;
+        }
+    }
+
+    @media(max-width: ${breakpoints.md}) {
+        flex-direction: column;
+        max-width: 60rem;
+        margin: 0 auto;
+    }
+
+    &:not(:last-child) {
+        margin-bottom: 10rem;
+    }
+
+    .promo__text {
+        width: 50%;
+        padding: 9rem 5rem;
+
+        @media(max-width: ${breakpoints.md}) {
+            width: 100%;
+            max-width: 60rem;
+        }
+    }
+
+    .promo__media {
+        position: relative;
+        width: 50%;
+        z-index: -1000;
+
+        @media(max-width: ${breakpoints.md}) {
+            width: 100%;
+            max-width: 60rem;
+        }
+
+        &::before {
+            content: "";
+            display: block;
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            background: #8c9bae;
+            z-index: 1000;
+            opacity: .1;
+        }
+    }
+
+    .promo__video {
```

Comment:
Why are there 2 underscores here?

Reply:
Thanks! :slightly_smiling_face: This is part of BEM a naming convention for naming classes in CSS. You can find out more about it here http://getbem.com/

Reply:
Sorry I'm still learning CSS :sweat_smile:

Reply:
Thanks! No Problem.